### PR TITLE
Merge testing into master

### PR DESCRIPTION
**Pull request by lapineige**
There are important changes pending is testing branch.

- Revamp of the app
- Add change url
- Update to 0.9.7
- Readme update

Should we ask for user feedback in the forum ? (Those changes work well on my side, but I'd prefer more tests before merging)

### CODE REVIEW

**Comment by anmol26s** [comment]
Change_url is not working?

File: README.md [hunk 1]
```diff
@@ -29,6 +28,8 @@ Maintainer: @Lapineige
 
 ## Installation information
 
+During the installation you will be prompted if you want to set this instance as public, which means any people (not just Yunohost users) can consult you Shaarli.
```
Comment (anmol26s) [inline README.md]: During the installation app will prompt you to install itself as public or private, if you choose public then anyone (not just Yunohost users) would be able to access your Shaarli instance.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
-# Shaarli app for Yunohost
-=======
-shaarli_ynh - Shaarli app for Yunohost
-===========
+# Shaarli app for Yunohost (shaarli_ynh)
+---
+
```
Comment (anmol26s) [inline README.md]: Shaarli app for Yunohost ~~(shaarli_ynh)~~
Its better to have short heading. Shaarli_ynh has no information for the end user.

**Comment by anmol26s** [comment]
It would be better if the Yunohost badge and integration level is placed just after the main heading.

**Comment by lapineige** [comment]
up :)

**Comment by lapineige** [comment]
@anmol26s I updated the readme :)

**Comment by anmol26s** [comment]
Looks good. Only the image in the readme is lost. But anyways we can merge it.

**Comment by lapineige** [comment]
Oh yeah the provider is unavailable for the moment.
Yeah anyway, this was an addition, it's not a big deal if it's not shown.
(I will change the provider if this takes too long)

**Comment by anmol26s** [comment]
Its been long time. We should merge this.

**Pull request merged**